index.html

```diff
@@ -196,5 +238,39 @@ <h1 class="fw5 f4">References</h1>
         </div>
       </div>
   </div>
+<script>
+document.querySelectorAll('button[data-copy]').forEach((button) => {
+  button.addEventListener('click', () => {
+    const codeId = button.getAttribute('data-copy');
+    const codeDiv = document.getElementById(codeId);
+    if (codeDiv && codeDiv.textContent && codeDiv.textContent.length > 0) {
+
+      // Create a temporary textarea to copy the text
+      const tempTextarea = document.createElement('textarea');
+      tempTextarea.value = codeDiv.textContent.trim();
+      document.body.appendChild(tempTextarea);
+      tempTextarea.select();
+      const copied = document.execCommand('copy');
+      document.body.removeChild(tempTextarea);
```

Comment:
`document.execCommand` is deprecated - should probably use [writeText](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText) instead?

Reply:
I'm totally fine with dropping IE11 - it's 13 years old now and doesn't support a wide array of features, including [arrow functions](https://caniuse.com/arrow-functions), so this won't be IE11 compatible anyway.